src/core/sys/dragonflybsd/sys/_cpuset.d

```diff
@@ -0,0 +1,30 @@
+/**
+ * D header file for DragonFlyBSD.
+ *
+ * Authors: Martin Nowak, Diederik de Groot(port:DragonFlyBSD)
+ * Copied:  From core/sys/freebsd/sys
+ */
+module core.sys.dragonflybsd.sys._cpuset;
+
+version (DragonFlyBSD):
+extern (C):
```

Comment:
Enums are not placed in an object file, so extern(C) makes no sense, think the same for aliases too?

Reply:
Remarked out 'extern(C):'